lib/process-services-cloud/src/lib/process/process-list/components/process-list-cloud.component.spec.ts

```diff
@@ -634,12 +600,12 @@ describe('ProcessListCloudComponent', () => {
             fixtureEmpty.destroy();
         });
 
-        it('should render the custom template', async () => {
+        it('should render the custom template', () => {
```

Comment:
Why removing the async, isn't the call spied here need to be fullfilled before?

Reply:
I see a mocked API call that probably needs to be fulfilled before the check, removing async/await whenstable means that we don't care about that call, is this the behavior we wanted?